src/main/java/core/basesyntax/handler/ConsoleHandler.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler implements Handler {
+    @Inject
+    private BetDao betDao;
+    @Dao
```

Comment:
```suggestion
    @Inject
```
There should be an `@Inject` annotation

Reply:
done